pyInterface/python_compiler.py

```diff
@@ -13,4 +13,5 @@
 
 	os.system('rm -rf ' + destdir + '/*')
 	os.system('cd ' + indir + '; for dir in `find * -type d`; do mkdir ' + destdir + '/$dir; done; cd - > /dev/null')
-	os.system('cd ' + indir + '; for pycfile in `find * -name "*.pyc"`; do mv $pycfile ' +  destdir + '/$pycfile; done; cd - > /dev/null')
+	os.system('cd ' + indir + '; for pycfile in `find . -name "*.pyc"`; do mv $pycfile ' +  destdir + '/$pycfile; done; cd - > /dev/null')
+	os.system('cd ' + indir + '; for pyfile in `find . -name "*.py"`; do cp $pyfile ' + destdir + '/$pyfile; done; cd - > /dev/null')
```

Comment:
Because pyLint asks for the .py files when checking the pyRootPwa package which comes from the build folder.